ansible/ci-build-collector-full.yml

```diff
@@ -46,7 +46,7 @@
             source: build
 
     - name: Retag slim image
-      when: not build_full_image or arch == 'arm64'
+      when: not build_full_image or arch == 'arm64' or arch == 'ppc64le' or arch == 's390x'
```

Comment:
revert before merging

Reply:
Not reverting - will be removed when the midstream is updated with this module version